hudi-spark-datasource/hudi-spark/src/test/scala/org/apache/spark/sql/hudi/procedure/TestClusteringProcedure.scala

```diff
@@ -50,31 +50,32 @@ class TestClusteringProcedure extends HoodieSparkProcedureTestBase {
              |  id int,
              |  name string,
              |  price double,
-             |  ts long
+             |  ts long,
+             |  partition long
              |) using hudi
              | options (
              |  primaryKey ='id',
              |  type = '$tableType',
              |  preCombineField = 'ts'
              | )
-             | partitioned by(ts)
+             | partitioned by(partition)
```

Comment:
why is there a need to change the table schema for test? Moreover, the new partition firld has the same type as the old one, so i'm not sure if it adds any value for the test.

Reply:
Using the partition field as precombine makes no sense and should not be done